src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -647,7 +649,9 @@ public override bool IsDefined(Type attributeType, bool inherit)
 
         public override bool IsGenericMethod { get { return m_inst != null; } }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
         public override Type[]? GetGenericArguments() { return m_inst; }
```

Comment:
This is documented to return empty array for non-generic types. Returning null is a bug. Do we track the bugs like this; or just fix them?

Reply:
In general we've just been fixing the ones we agree to fix, opening issues for ones we think we might want to but need discussion, and ignoring the rest (e.g. places where we null ref instead of throwing ArgumentNullException and it's sufficiently corner-case to not bother adding argument validation / paying the cost to do so).

Reply:
I guess this one makes sense to fix here?

Reply:
I fixed it here.